ngraph/python/tests/test_onnx/utils/model_importer.py

```diff
@@ -26,7 +26,16 @@
 from onnx.backend.test.runner import TestItem
 from pathlib import Path
 from tests.test_onnx.utils.onnx_helpers import import_onnx_model
-from typing import Any, Dict, List, Optional, Pattern, Set, Text, Type, Union
+from typing import Any, Dict, List, Optional, Pattern, Set, Text, Type, Union, Callable, Sequence
+
+
+class ExtOnnxTestCase(OnnxTestCase):
+    def __new__(cls, name, model_name, url, model_dir, model, data_sets, kind, rtol, atol, post_processing):
```

Comment:
Use `__init__` instead of `__new__` and set `post_processing=None` by default.

Reply:
I changed way of `ExtOnnxTestCase` creation to use `_fields`